alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -326,7 +326,7 @@ class AlertIndices(
         }
         val findingsIndex = dataSources.findingsIndex
         val findingsIndexPattern = dataSources.findingsIndexPattern ?: FINDING_HISTORY_INDEX_PATTERN
-        if (!clusterService.state().metadata().hasAlias(findingsIndexPattern)) {
+        if (!clusterService.state().metadata().hasAlias(findingsIndex)) {
```

Comment:
Why do we need to set the ``findingsIndexPattern`` here?

Reply:
We're using findingsIndexPattern as index name in createIndex method right below.